cmds/cpu/doc.go

```diff
@@ -5,116 +5,122 @@
 //
-//     CPU has many options, as shown above; most you need not worry about.
-//     The most common invocation is
-//     cpu -h hostname
-//     which will start a shell and mount the 9p server in /tmp/cpu.
-//     Note this mount proceeds over the ssh session, and further
-//     it mounts in a private /tmp; there is little to see when
-//     it is running from outside the ssh session
+//	CPU has many options, as shown above; most you need not worry about.
+//	The most common invocation is
+//	cpu -h hostname
+//	which will start a shell and mount the 9p server in /tmp/cpu.
+//	Note this mount proceeds over the ssh session, and further
+//	it mounts in a private /tmp; there is little to see when
+//	it is running from outside the ssh session
 //
 // Options:
-//     -9p bool
-//           enable the 9p server in the client (default enabled)
-//           Note that the 9p server is also enabled if the namespace
-//           is not empty. To ensure the 9p server is not set,
-//           cpu -9p=f -namespace=""
-//     -d
-//           enable debug prints
-//     -dbg9p
-//           show 9p io
-//     -dump
-//           Dump all debug output and 9p packets to a file in /tmp
-//     -hk string
-//           host key file
-//     -key string
-//           key file (default "$HOME/.ssh/cpu_rsa")
-//     -mountopts string
-//           extra options for the 9p mount, default "". Lightly tested.
-//     -msize uint
-//           max size for 9p packets, default 1 MiB
-//     -namespace string
-//           namespace defines the bind mounts that are done by cpud.
-//           The format is of a : separated string, in the style of PATH
-//           variables, with one addition: one can define where on the
-//           host the bind is done.
-//           The default is /lib:/lib64:/usr:/bin:/etc:/home, which means
-//           these directories will be provided by the 9p server in the
-//           client. Sometimes, it is useful to be able to remap the names.
-//           On OSX, for example, /Users/rob would be rob's home directory,
-//           but we might want it to appears as /home/rob on Linux.
-//           This change is accomplished with
-//           -namespace /lib:/lib64:/usr:/bin:/etc:/home/rob=/Users/rob
-//     -network string
-//           network to use (default "tcp")
-//     -port9p string
-//           port9p # on remote machine for 9p mount
-//     -remote
-//           Indicates we are the remote side of the cpu session
-//     -root
-//           Root for 9p server, default "/"
-//           If you are cpu'ing from, eg., x86 to arm, you might
-//           use, e.g., /amd64
-//     -sp string
-//          remote port, default 17010
-//     -srv string
-//           what server to run (default none; use internal)
-//     -timeout9p time.Duration
-//           How long to wait for the server to connect to 9p (default100ms)
+//
+//	-9p bool
+//	      enable the 9p server in the client (default enabled)
```

Comment:
Weren't there words above about how that this is no longer the case?

Reply:
fixed. Thanks. Good catch.